src/components/dictionaryConcepts/components/ConceptDropdown.jsx

```diff
@@ -35,28 +35,28 @@ const ConceptDropdown = props => (
         Create new concept
       </button>
       <div className="dropdown-menu" aria-labelledby="dropdownMenuButton">
-        <Link className="dropdown-item" to={`${props.pathName}/new/diagnosis`}>
+        <Link className="dropdown-item" to={`/new${props.pathName}/diagnosis`}>
```

Comment:
Fix all these links. Add a slash after new

Reply:
There shouldn't be. The backslash is already there in the pathname.